src/Config/RuntimeConfigLoader.cs

```diff
@@ -22,15 +22,32 @@ namespace Azure.DataApiBuilder.Config;
 
 public abstract class RuntimeConfigLoader
 {
+    public HotReloadEventHandler<CustomEventArgs>? Handler;
     protected readonly string? _connectionString;
 
     // Public to allow the RuntimeProvider and other users of class to set via out param.
     // May be candidate to refactor by changing all of the Parse/Load functions to save
     // state in place of using out params.
     public RuntimeConfig? RuntimeConfig;
 
-    public RuntimeConfigLoader(string? connectionString = null)
+    public RuntimeConfig? usedRuntimeConfig;
```

Comment:
Added line 33

Reply:
"usedRuntimeConfig" is the old runtime config that is already parsed and validated, the purpose is to use it in the case that the new config is not able to be parsed. So the user will receive a message that the new config is invalid, but they would be able to still use dab with the old config configuration.
I will change the name to "oldRuntimeConfig" or "validatedRuntimeConfig" so it doesn't cause confusion